tests/test_vlan.py

```diff
@@ -447,13 +447,10 @@ def test_AddPortChannelToVlan(self, dvs, testlog):
         assert len(lag_member_entries) == 1
 
         (status, fvs) = tbl.get(lag_member_entries[0])
-        for fv in fvs:
-            if fv[0] == "SAI_LAG_MEMBER_ATTR_LAG_ID":
-                assert fv[1] == lag_entries[0]
-            elif fv[0] == "SAI_LAG_MEMBER_ATTR_PORT_ID":
-                assert dvs.asicdb.portoidmap[fv[1]] == "Ethernet0"
-            else:
-                assert False
+        fvs = dict(fvs)
```

Comment:
There are multiple places in the file use the original ways to get the field/values, should we keep them consistent?   If there are more fields, we probably could check other fields too, or if we just wanted to ignore them, remove the "else" branch.

Reply:
I think this is the more cleaner and efficient way of checking field value match. I do NOT want to cleanup at other places as part of this PR.

Yes, in this case we do NOT care other (two) fields, removing else would have solved the problem, so does the new code and it much cleaner and efficient! So, I would prefer to keep it as is.

Reply:
OK, I saw master has similar fix,  it should be fine.